src/components/editor/PropertyResourceTemplate.jsx

```diff
@@ -12,50 +12,63 @@ class PropertyResourceTemplate extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      collapse: false
+      collapse: false,
+      output: []
     }
+    this.populatePropertyTemplates()
   }
 
   handleAddClick = (event) => {
-     event.preventDefault()
+    event.preventDefault()
+    const existingOutputs = this.state.output
+    existingOutputs.push(<h4 key={shortid.generate()}>{this.props.resourceTemplate.resourceLabel}</h4>)
+    this.populatePropertyTemplates()
+    this.setState({ output: existingOutputs})
   }
 
   handleMintUri = (event) => {
     event.preventDefault()
   }
 
+  populatePropertyTemplates = () => {
+    this.props.resourceTemplate.propertyTemplates.map((property) => {
+      let keyId = shortid.generate()
+      let newReduxPath = Object.assign([], this.props.reduxPath)
+      newReduxPath.push(keyId)
+      this.state.output.push(<PropertyTemplateOutline
+              propertyTemplate={property}
+              rtId={this.props.resourceTemplate.id}
+              reduxPath={newReduxPath}
+              key={keyId} />)
+    })
+  }
+
   render() {
+    const isAddDisabled = this.props.isRepeatable == "false" ? true : false
```

Comment:
This is a holdover from parsing the JSON Resource Template created in the Profile Editor, the PE sets these boolean "false" & "true" as a string values. Y

Reply:
Aha, OK. Thanks, @jermnelson.

Reply:
If I'm reading this code correctly, I think this ternary will give us the wrong answer if we get a resourceTemplate that uses boolean (not string) `true`, which is legit per schemas.  We need an easy way to get boolean `true` as a result for both `"true"` and `true` while similarly getting boolean `false` for `"false"` and `false`. Maybe a util method to convert string "boolean" to actual boolean values and then just use whatever it returns?

Reply:
Also, since Mike and I both got a little puzzled by this, and it appears more than once ... would that imply a well named util function would be better, even if I'm mistaken about it possibly giving wrong answer?

Reply:
I really dig this idea. And :+1: to wrapping values in `Boolean()` in said util function.

Reply:
See latest commit with the `isRepeatable` function from Utilities module.